metecho/api/tests/admin.py

```diff
@@ -42,7 +42,7 @@ def test_queryset__deleted(self, epic_factory):
 
         args = (
             None,
-            {"deleted_at": "true"},
+            {"deleted_at": ["true"]},
```

Comment:
We can set this aside until we do move to Django 5x:
```suggestion
            {"deleted_at": "true"},
```